src/main/kotlin/com/cognifide/gradle/aem/common/instance/service/workflow/WorkflowManager.kt

```diff
@@ -73,8 +74,13 @@ class WorkflowManager(sync: InstanceSync) : InstanceService(sync) {
 
                 while (stack.isNotEmpty()) {
```

Comment:
this is brute-forcing mechanism for restoring ASAP workflow launcher just after deploying package